### PR TITLE
Last updates from 02-10-2020

### PR DESCRIPTION
**Pull request by agstod**
Last updates based on Vespert's comprehensive list of changes on BGG

### CODE REVIEW

File: src/assets/rules.yml [hunk 1]
```diff
@@ -495,9 +495,7 @@
                 - name: Attacking with Ally.
                   text: Whenever you initiate a battle, you may treat Allied warriors of one faction in the clearing of battle as your own. The maximum hits you can roll equals the number of that Ally's warriors there plus your total undamaged `item:sword`. You cannot treat Allied warriors as your own if that Ally is the defender.
                 - name: Taking Hits with Ally.
-                  text: In a battle where you treat Allied warriors as your own, you can take hits by removing Allied warriors. However, if you take more hits by removing Allied warriors than by damaging items during the same battle, that Allied faction becomes Hostile (`rule:9.2.9.3`). This rule overrides the condition for becoming Hostile in 9.2.9.III.
-                - name: Outrage with Ally.
```
Comment (seiyria) [inline src/assets/rules.yml]: This rule was removed? Or is it explained somewhere else now?

**Comment by seiyria** [comment]
Just had one question but looks good!

**Comment by agstod** [comment]
I know! Straight up removed! 

It actually caused me more confusion because they said that 1.1.4 covers it now...

**Comment by seiyria** [comment]
That's weird. If it's redundant, sure, by all means. Thanks again!

**Pull request merged**